Automattic/jetpack
# Masterbar: Disable Nav Unification when SSO is inactive

Fixes https://github.com/Automattic/wp-calypso/issues/52921

#### Changes proposed in this Pull Request:

When SSO is disabled, users who log in into WP Admin are confused when they are asked to log in into Calypso after they click on a menu item that links to Calypso.

To improve the experience for these users, this PR disables Nav Unification on WP Admin when SSO is inactive.

#### Jetpack product discussion
N/A.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Install Jetpack Beta on an Atomic site and switch to the branch of this PR.
* Go to Settings > General > Security
* Disable SSO by turning off the "Allow users to log in to this site using WordPress.com accounts" setting.
* Go to WP Admin.
* Make sure you get the default WP Admin menu, with no nav unification changes.
* Make sure there are no changes when SSO is enabled.
